package-lock.json

```diff
@@ -2068,7 +2070,6 @@
         "deepmerge": "3.2.0",
         "dispensary": "0.37.0",
         "es6-promisify": "6.0.1",
-        "eslint": "5.16.0",
```

Comment:
@Rob--W noticed that in the package-lock.json file renovatebot is removing the explicit eslint dependency from our addons-linter dependency section.